plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -470,8 +481,25 @@ public List<ExecutorBuilder<?>> getExecutorBuilders(Settings settings) {
             ML_THREAD_POOL_PREFIX + PREDICT_THREAD_POOL,
             false
         );
+        FixedExecutorBuilder reloadModelThreadPool = new FixedExecutorBuilder(
```

Comment:
We don't need to create a thread pool to accomplish this, we can reuse the general thread pool in opensearch or even a simple Thread would be sufficient.

Reply:
yes, I have removed it